### ISSUE TITLE
Check that regenerating the test manifest does not change it

### ISSUE THREAD

**Issue by jdm** [servo/servo#8587]
We have a check that runs on TravisCI that ensures that our Cargo.lock files do not change as part of the build: https://github.com/servo/servo/blob/master/etc/ci/lockfile_changed.sh
We should run a similar check that `./mach test-wpt --manifest-update` does not change either of the `MANIFEST.json` files under `tests/wpt`. This requires adding a script to this repository, then adding it to the [list of commands](https://github.com/servo/saltfs/blob/master/buildbot/master/master.cfg#L190) to run on the build machines.

This can be tested locally by adding a new test to `tests/wpt/mozilla/tests/mozilla` and making sure the new script reports a failure.

**Comment by mfeckie** [comment]
@jdm I'm keen to try this one

**Comment by jdm** [comment]
Please do!

**Comment by mfeckie** [comment]
So, just ran the tests for the first time and they changed the manifest :disappointed: 

Looking at it the first one may be insignificant / related to my system.  The second change looks like it might be legitimate, but am unfamiliar.  @jdm or delegate, could you advise?

```
--- a/tests/wpt/metadata/MANIFEST.json
+++ b/tests/wpt/metadata/MANIFEST.json
@@ -35511,4 +35511,4 @@
   "rev": "5a754b40cd49c0404863c431b58cc311dc5d167c",
   "url_base": "/",
   "version": 2
-}
\ No newline at end of file
+}
diff --git a/tests/wpt/mozilla/meta/MANIFEST.json b/tests/wpt/mozilla/meta/MANIFEST.json
index fb9483e..4b1e4e3 100644
--- a/tests/wpt/mozilla/meta/MANIFEST.json
+++ b/tests/wpt/mozilla/meta/MANIFEST.json
@@ -9441,7 +9441,7 @@
           "path": "css/viewport_ignore_desktop.html",
           "references": [
             [
-              "/_mozilla/css/viewport_ignore_desktop.html",
+              "/_mozilla/css/viewport_ignore_desktop_ref.html",
               "=="
             ]
           ],
(END)
```

**Comment by jdm** [comment]
The first is unimportant like you say, but is likely caused by someone's editor while they hand-edited it for some mysterious reason. The second looks like a legitimate problem that this script is designed to catch :)

**Comment by jdm** [comment]
That is, I suspect that test was added by hand incorrectly and we should fix it.

**Comment by Wafflespeanut** [comment]
fixed by #8601
